modules/site-health/audit-enqueued-assets/helper.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Helper functions used by module.
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Gets total of enqueued scripts.
+ *
+ * @since 1.0.0
+ *
+ * @return int|false Number of total scripts or false if transient hasn't been set.
+ */
+function perflab_aea_get_total_enqueued_scripts() {
+	$enqueued_scripts      = false;
+	$list_enqueued_scripts = get_transient( 'aea_enqueued_front_page_scripts' );
+	if ( $list_enqueued_scripts ) {
+		$enqueued_scripts = count( $list_enqueued_scripts );
+	}
+	return $enqueued_scripts;
+}
+
+/**
+ * Gets total size in bytes of Enqueued Scripts.
+ *
+ * @since 1.0.0
+ *
+ * @return int|false Byte Total size or false if transient hasn't been set.
+ */
+function perflab_aea_get_total_size_bytes_enqueued_scripts() {
+	$total_size            = false;
+	$list_enqueued_scripts = get_transient( 'aea_enqueued_front_page_scripts' );
+	if ( $list_enqueued_scripts ) {
+		foreach ( $list_enqueued_scripts as $enqueued_script ) {
+			$total_size += $enqueued_script['size'];
+		}
+	}
+	return $total_size;
+}
+
+/**
+ * Gets total of enqueued styles.
+ *
+ * @since 1.0.0
+ *
+ * @return int|false Number of total styles or false if transient hasn't been set.
+ */
+function perflab_aea_get_total_enqueued_styles() {
+	$enqueued_styles      = false;
+	$list_enqueued_styles = get_transient( 'aea_enqueued_front_page_styles' );
+	if ( $list_enqueued_styles ) {
+		$enqueued_styles = count( $list_enqueued_styles );
+	}
+	return $enqueued_styles;
+}
+
+/**
+ * Gets total size in bytes of Enqueued Styles.
+ *
+ * @since 1.0.0
+ *
+ * @return int|false Byte Total size or false if transient hasn't been set.
+ */
+function perflab_aea_get_total_size_bytes_enqueued_styles() {
+	$total_size           = false;
+	$list_enqueued_styles = get_transient( 'aea_enqueued_front_page_styles' );
+	if ( $list_enqueued_styles ) {
+		foreach ( $list_enqueued_styles as $enqueued_style ) {
+			$total_size += $enqueued_style['size'];
+		}
```

Comment:
I think it's safer to set `$total_size` to `0` before starting to add to it. I think this will work, but it's technically problematic, and I could see how this may eventually cause a warning (you shouldn't add numeric values to a boolean).
```suggestion
	if ( $list_enqueued_styles ) {
		$total_size = 0;
		foreach ( $list_enqueued_styles as $enqueued_style ) {
			$total_size += $enqueued_style['size'];
		}
```

Reply:
good catch.